package.json

```diff
@@ -6,7 +6,9 @@
   "scripts": {
     "build": "webpack",
     "lint": "eslint src",
-    "test": "jest"
+    "test": "jest '.*.spec.ts'",
+    "test-server": "cd e2e && webpack serve",
```

Comment:
Dev server is probably fine for now, we may want to swap this out for a full build / node serve in the future. The dev server isn't super performant

Reply:
This is just here for simplicity, so that playwright has an easy way to start a dev server. That's all it's used for, just building the JS and serving the html for the e2e tests, and we don't have any performance concerns there (it gets hit about 5 times during tests!).

Reply:
I have added a file-level comment about what this webpack config is for.